packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -106,40 +111,39 @@ export const announceApplications = async ({
   router: VueRouter
   translations: unknown
   supportedLanguages: { [key: string]: string }
-}): Promise<void> => {
+}): Promise<any> => {
```

Comment:
the application initialization happens now in the index.ts and also needs to wait in case if something errors. Besides that please do not use the TS any type

Reply:
yes exactly